src/core-components/Header/Header.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { Meta, Story } from '@storybook/react';
+import { Col, Row } from 'antd';
+import React from 'react';
+
+import { Header } from './Header';
+
+export default {
+  title: 'Components/Header',
+  component: Header,
+} as Meta<typeof Header>;
+
+export const Template: Story = () => (
```

Comment:
Add `args` to this story. So that we will be able to manage the state on the story page.